### PR TITLE
Create a wall

### PR DESCRIPTION
**Pull request by samyakbvs**
I have added this task as most of the tiles have the same walls and with same walls in every tile the game will look monotonous

### CODE REVIEW

**Comment by samyakbvs** [comment]
@niccokunzmann

File: tasks/Create-A-wall.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
+#Labyrinth: Create a Wall
+
+The [FOSSASIA Labyrinth](https://github.com/fossasia/labyrinth/) allows you to contribute parts to a huge labyrinth.
+We can add new features to the labyrinth which improve the game.
+
+Requirements:
+- Download and install Inkscape 
+- Create a Wall for the Labyrinth maze
```
Comment (niccokunzmann) [inline tasks/Create-A-wall.md]: "Wall" -> "wall"

File: tasks/Create-A-wall.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
+#Labyrinth: Create a Wall
+
+The [FOSSASIA Labyrinth](https://github.com/fossasia/labyrinth/) allows you to contribute parts to a huge labyrinth.
+We can add new features to the labyrinth which improve the game.
+
+Requirements:
+- Download and install Inkscape
```
Comment (niccokunzmann) [inline tasks/Create-A-wall.md]: Would you like to have a link to the download site of Inkscape here?

File: tasks/Create-A-wall.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+- Download and install Inkscape 
+- Create a Wall for the Labyrinth maze 
+- Add your wall to labyrinth/tiles/rooms/wall
+- A merged pull request
+
+Expected outcome:
+- Added atleast one wall
+- One pull requset is succesfully merged
+
+Hints:
```
Comment (niccokunzmann) [inline tasks/Create-A-wall.md]: I would add a hint that you can start with an existing wall or tile to get the placing and the dimensions.

File: tasks/Create-A-wall.md [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+- You can use techniques like [pair programming](https://www.youtube.com/watch?v=vgkahOzFH2Q) or collaborate together on branches of your forks. You can attribute a pull-request also to other people who helped you out or submit a pull-request where you and an other person contributed together.
+- You can join our [chat on gitter](https://gitter.im/fossasia/labyrinth) if you get stuck to get help or just hang out and have a nice time with others.
+
+---
+
+Categories:
+- [X] Code
+- [X] User Interface
+- [ ] Documentation / Training
+- [X] Quality Assurance
```
Comment (niccokunzmann) [inline tasks/Create-A-wall.md]: Is this quality assurance in your eyes? If yes, all fine!

File: tasks/Create-A-wall.md [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+We can add new features to the labyrinth which improve the game.
+
+Requirements:
+- Download and install Inkscape 
+- Create a Wall for the Labyrinth maze 
+- Add your wall to labyrinth/tiles/rooms/wall
+- A merged pull request
+
+Expected outcome:
+- Added atleast one wall
```
Comment (niccokunzmann) [inline tasks/Create-A-wall.md]: add one space

File: tasks/Create-A-wall.md [hunk 6]
```diff
@@ -0,0 +1,27 @@
...
+- One pull requset is succesfully merged
```
Comment (niccokunzmann) [inline tasks/Create-A-wall.md]: I would add that the wall should be part of the labyrinth. And that issues based on the created pr must be solved.

File: tasks/Create-A-wall.md [hunk 7]
```diff
@@ -0,0 +1,27 @@
+#Labyrinth: Create a Wall
+
+The [FOSSASIA Labyrinth](https://github.com/fossasia/labyrinth/) allows you to contribute parts to a huge labyrinth.
+We can add new features to the labyrinth which improve the game.
```
Comment (niccokunzmann) [inline tasks/Create-A-wall.md]: We can use your issue statement here: 
"As most of the tiles have the same walls and with same walls in every tile the game will look monotonous, please add more walls!"

**Comment by xeon-zolt** [comment]
@niccokunzmann it better to create changes here only

**Comment by samyakbvs** [comment]
Okay

File: tasks/Create-A-wall.md (hunk 4 above)
Comment (samyakbvs) [reply tasks/Create-A-wall.md]: Yes it is I think

**Comment by samyakbvs** [comment]
@niccokunzmann Pls merge when you get free

**Pull request merged**